resources/texts/change_log.txt

```diff
@@ -1,3 +1,72 @@
+[5.7]
+* New features and improvements:
```

Comment:
Can you add a blank line here to keep consistency with other versions

Reply:
[d42f894](https://github.com/Ultimaker/Cura/pull/18615/commits/d42f894965fba80f1f4fad47b957e09a071509e2)